package.json

```diff
@@ -30,11 +30,13 @@
   },
   "dependencies": {
     "cron-master": "~0.2.0",
+    "env-var": "~2.4.3",
     "later": "~1.2.0",
     "moment": "~2.17.1",
+    "rhmap-raygun-nodejs": "git+https://github.com/TkeITMobility/rhmap-raygun-nodejs.git",
```

Comment:
When I have `"rhmap-raygun-nodejs": "git+https://github.com/TkeITMobility/rhmap-raygun-nodejs.git#v0.2.0"` in there, `npm install` gives me this guy 'fatal: ambiguous argument 'v0.2.0': unknown revision or path not in the working tree.'  Any thoughts on that?  I've been pulling my hair